geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/ExecuteRegionFunctionSingleHop.java

```diff
@@ -307,9 +307,11 @@ public void cmdExecute(final Message clientMessage, final ServerConnection serve
           resultSender.setException(fe);
         }
       } else {
-        logger.warn(LocalizedMessage.create(
-            LocalizedStrings.ExecuteRegionFunction_EXCEPTION_ON_SERVER_WHILE_EXECUTIONG_FUNCTION_0,
-            function), fe);
+        if (logger.isDebugEnabled()) {
```

Comment:
This to me feels like there is a disconnect between the commit message and the actual change. 
The commit message makes it sound like there are multiple errors being logged and we are addressing the fact that there are multiple. The actual change though just changes the log level from warn to debug.

Another stupid question: how comes we have to check the log level in an if-statement? Shouldn't we get this for free from `logger.debug`?

Reply:
@pivotal-amurmann the error is called "multiple node found" which is one error. When rebalance etc. are happening we may encounter exceptions like multiple target nodes errors, bucket moved exception etc. Function execution knows that for these kind of errors it needs to re-attempt the execution. But it will confuse a end user if they suddenly see random appearance of these exceptions in the logs.

Reply:
Thanks for the explanation. I wonder if the commit message could have explained this better.